src/qawsiotnetworkrequest_p.h

```diff
@@ -15,10 +15,10 @@ class QTAWSIOT_AUTOTEST_EXPORT QAwsIoTNetworkRequest: public QMqttNetworkRequest
                           const QString &sessionToken);
     virtual ~QAwsIoTNetworkRequest() = default;
     QAwsIoTNetworkRequest(const QAwsIoTNetworkRequest &) = default;
-    QAwsIoTNetworkRequest(QAwsIoTNetworkRequest &&) = default;
+    //QAwsIoTNetworkRequest(QAwsIoTNetworkRequest &&) = default;
```

Comment:
I would rather implement this or #ifdef this than comment this out.
It has the advantage that on real compilers this would work then.